### ISSUE TITLE
OS Open Basemaps unavailable

### ISSUE THREAD

**Issue by FsenyahPHE** [doug-m/CollectorAuroraTesting#129]
### Issue
I'm working in the UK, and can see a number of OS Open (Ordnance Survey) basemaps in the map gallery which could be of use.  However they are all greyed out under the header of 'UNAVAILABLE' and clicking on any of them brings up the error 'Basemap Unavailable' - The spatial reference is not compatible with the map.  Is this a restriction on the basemap or some other error?

### Expected Behavior
When clicking the Basemap option in the App, OS Open basemaps should be selectable

### Map and Credentials
N/A

### Steps to Reproduce
Click Basemaps in the Basemap Gallery, under the options, OS Open basemaps should be selectable 

### Device Information
IPhone 7
Version 11.4

![osopen](https://user-images.githubusercontent.com/40383669/41588721-7172c2d8-73aa-11e8-8d5f-42a114949a26.jpg)

**Comment by doug-m** [comment]
@FsenyahPHE in the current version of Collector are you working with the map offline?

**Comment by FsenyahPHE** [comment]
@doug-m Yes it works fine currently both online and offline in current Collector,I've just re-tested.

**Comment by doug-m** [comment]
@FsenyahPHE I suspect that you've got additional basemaps shared within your organization.  Could you share the specific itemids for the OS basemaps that are part of your organization so we can add them to ours?
